spec/features/residents_can_see_confirmation_of_repair_request_spec.rb

```diff
@@ -28,12 +29,22 @@
 
     # Address selection:
     choose_radio_button 'Ross Court 30'
-    click_on 'Use this address'
+    click_on 'Continue'
 
     # Contact details - last page before confirmation:
+    fill_in 'Full name', with: 'John Smith'
+    fill_in 'Telephone number', with: '01234567890'
+    check 'afternoon (12pm - 5pm)'
     click_on 'Continue'
 
     expect(page).to have_content 'Your reference number is abc123'
+
+    expect(page).to have_content "Description: The bath won't drain"
+
+    expect(page).to have_content 'Name: John Smith'
+    expect(page).to have_content 'Phone: 01234 567 890'
     expect(page).to have_content 'Address: Ross Court 30, E5 8TE'
+
+    expect(page).to have_content 'Time: afternoon (12pm - 5pm)'
```

Comment:
I think we should have a test for what happens when both callback times are checked, as I'm not sure what the output should look like.

Reply:
Yup - that's covered later in the `Confirmation` presenter object: the approach I took was to test specifics in that object, and then have just one feature spec which tests the integration.

Reply:
:+1:

I did spot this after I posted my previous comment, but then forgot to go back and remove it!